htdocs/js/entities.js

```diff
@@ -5,7 +5,7 @@
  * @author Justin Otherguy <justin@justinotherguy.org>
  * @author Steffen Vogel <info@steffenvogel.de>
  * @author Andreas Götz <cpuidle@gmx.de>
- * @copyright Copyright (c) 2011, The volkszaehler.org project
+ * @copyright Copyright (c) 2011-2017, The volkszaehler.org project
  * @package default
```

Comment:
remove package?

Reply:
Oh, ok. I focused on PHP files only, because thought, that htdocs/js/*.js are generated javascript files. I will change it.